sandcastle/api.py

```diff
@@ -450,8 +450,8 @@ def _do_exec(
             except ApiException as ex:
                 # in packit-service prod, occasionally 'No route to host' happens here
                 # let's try to repeat the request
-                logger.error("exception while initiating WS Client: %r", ex)
-                time.sleep(i + 1)
+                logger.error("exception while initiating WS Client: %s", ex)
+                time.sleep(2 * i + 1)
```

Comment:
[better late than never](https://github.com/packit-service/sandcastle/pull/36#discussion_r333900120)

Reply:
so do we want it to be exponential? I was considering it, but on the other hand, I want the last tick to be 30sec max

Reply:
It's OK as it is.